CollectionView/ViewController.swift

```diff
@@ -16,7 +16,53 @@ class ViewController: UIViewController {
     
     override func viewDidLoad() {
```

Comment:
I can't comment on line 14, but the variable name is misleading since the object is an instance of `UIImage` and not  `UIImageView`. Also, this instance is not used in the view controller, but in the cell. I think a better approach would be just passing in the name of the image to the cell and then the cell would create the `UIImage` object.

Reply:
That's right. 
Fixed now